modules/hcp-dc-mgmt-cluster.adoc

```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy-disconnected.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dc-mgmt-cluster_{context}"]
+= Setting up a management cluster for {hcp} in a disconnected environment
+
+To set up an {product-title} management cluster, you can use dev-scripts, or if you are based on virtual machines, you can use the `kcli` tool. The following instructions are specific to the `kcli` tool.
```

Comment:
I'm also unsure about this whole section when using Openshift Virtualization.
@davidvossel ?

Reply:
I moved this to a bare-metal-specific section of the disconnected docs.